types/retry/options.d.ts

```diff
@@ -0,0 +1,66 @@
+export namespace Options {
```

Comment:
Don't make a single top-level namespace like this - it doesn't do anything useful. Unwrap and put everything as top-level `export`s.